src/game-support/path-of-exile2.md

```diff
@@ -0,0 +1,14 @@
+# Path of Exile 2
+<!-- script:Aliases [] -->
+
+{{#template ../templates/rating.md status=Silver installs=Yes opens=Yes}}
+
+When using DirectX11 or Vulkan renderer it produces around 40 FPS on any settings (M1 Max, 3440x1440)
+The game sometimes stutters in normal locations and has significant FPS problems in boss fights.
```

Comment:
Please mark it under the > [WARNING] label. I would consider downgrading the status to Bronze as you mentioned significant problems sometimes.

Reply:
Marked as warning and changed it to Bronze.